### PR TITLE
Release 0.2.0 branch

### PR DESCRIPTION
**Pull request by stefanoborini**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,4 +37,8 @@ def write_version_py():
         ]
     },
     packages=find_packages(),
+    install_requires=[
+        "force_bdss >= 0.2.0",
+        "granta"
```
Comment (jjenthought) [inline setup.py]: Is it checking if the plugin is already installed here?

Comment (stefanoborini) [reply setup.py]: no, this is their own library. We can't distribute that because it's closed source, so we just check for the dependency.

**Comment by stefanoborini** [comment]
Right... so we have a problem with this going green because of course it does not find the granta library.

**Comment by jjenthought** [comment]
Ah I see. I suppose there’s not a lot we can do about that as far as CI goes

**Review by jjenthought — APPROVED** [review]

**Comment by stefanoborini** [comment]
@jjenthought there is, I am going to write a dummy granta module

**Pull request merged**